src/Tests/Topology/When_configuring_publishers.cs

```diff
@@ -34,7 +34,7 @@ public void Should_not_map_publisher_and_type_if_it_is_not_a_message()
 
             var exception = Assert.Throws<InvalidOperationException>(() => configuration.GetPublishersFor(typeof(MyType)));
             StringAssert.Contains($"No publishers configured for `{typeof(MyType).FullName}`", exception.Message);
-            StringAssert.Contains($"Please add mappings via `endpointConfiguration.UseTransport<AzureServiceBusTransport>().UseTopology<EndpointOrientedTopology>().RegisterPublisherForType(\"endpointname\", typeof({typeof(MyType).FullName}))`", exception.Message);
+            StringAssert.Contains($"Register mappings via `endpointConfiguration.UseTransport<AzureServiceBusTransport>().UseTopology<EndpointOrientedTopology>().RegisterPublisherForType(\"endpointname\", typeof({typeof(MyType).FullName}))`", exception.Message);
```

Comment:
Is that what you want to test here? The name of the test implies that nothing should happen if registered type is not a message. If the intent is to validate that given it was the only type registered, therefore exception is thrown, perhaps just test for exception, not the exception message?